src/main/res/values-de/strings.xml

```diff
@@ -0,0 +1,164 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+  <string name="action_failed">Aktion fehlgeschlagen</string>
+  <string name="add_repository">Paketquelle hinzufügen</string>
+  <string name="address">Addresse</string>
```

Comment:
```suggestion
  <string name="address">Adresse</string>
```

Reply:
Correct spelling in german is indeed with one d. 🤔